pkg/asset/installconfig/libvirt/libvirt.go

```diff
@@ -22,13 +22,27 @@ func Platform() (*libvirt.Platform, error) {
 			},
 			Validate: survey.ComposeValidators(survey.Required, uriValidator),
 		},
-	}, &uri)
-	if err != nil {
+	}, &uri); err != nil {
+		return nil, errors.Wrap(err, "failed UserInput")
+	}
+
+	var poolPath string
```

Comment:
Do not add this as a question for the user. This does not meet the bar for inclusion in the guided-install path.